### ISSUE TITLE
broadband access

### ISSUE THREAD

**Issue by atperzynski** [NikKrieger/sociome#12]
Feature request: 
Add ability to include broadband internet in/with the ADI. Would be best to simply allow users to add "wildcard" variables from ACS that would be downloaded into the file.

Relevant table for broadband access is B28002. Here any user would likely need two values the household total and then the total with the attribute (e.g. broadband of any type) to get a %. Users would simply be able to list the variables and sociome would retrieve those for the relevant geography and year. 

At this point I think it is premature to consider including such additional variables in the ADI as a new function, rather users would have to do that work on their own. 

Variable | Label
-- | --
B28002001 | Total:
B28002002 | With An Internet Subscription
B28002003 | Dial-Up With No Other Type Of Internet Subscription
B28002004 | Broadband Of Any Type
B28002005 | Cellular Data Plan
B28002006 | Cellular Data Plan With No Other Type Of Internet Subscription
B28002007 | Broadband Such As Cable, Fiber Optic Or Dsl
B28002008 | Broadband Such As Cable, Fiber Optic Or Dsl With No Other Type Of Internet Subscription
B28002009 | Satellite Internet Service
B28002010 | Satellite Internet Service With No Other Type Of Internet Subscription
B28002011 | Other Service With No Other Type Of Internet Subscription
B28002012 | Internet Access Without A Subscription
B28002013 | No Internet Access

**Comment by NikKrieger** [comment]
Would it fulfill your request to add in another argument called `other_vars`? The user would pass a vector of the names of the desired variables, like `get_adi(..., other_vars = c("B28002001", "B28002006", "B28002007"))`. And the values of those variables be tacked onto the end of the output? But the ADI column itself would be unchanged?

**Comment by atperzynski** [comment]
Yes that should cover how people would use this 99.9% of the time.


Adam T. Perzynski, PhD
Chief Science and Strategy Officer
Global Health Metrics
*Staying at home *
Cleveland, OH
Cell: 216-509-3283
adam.perzynski@globalhealthmetrics.com
